### PR TITLE
fix(bundle-size-tools): Handle chunks in multiple assets

### PR DESCRIPTION
**Pull request by tylerbutler**
The previous code assumed that an asset only contained a single chunk. That is not always true, so I fixed the code to filter all assets that contain a given chunk, then sum them all instead of returning a single asset's size. When multiple assets contain a chunk, a warning is logged to help us determine if that's expected or not.

I also cleaned up some logging that was confusing while debugging.

Related to [AB#4457](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/4457)

### CODE REVIEW

File: build-tools/packages/bundle-size-tools/src/utilities/getChunkParsedSize.ts [hunk 1]
```diff
@@ -14,19 +14,29 @@ export function getChunkParsedSize(stats: StatsCompilation, chunkId: string | nu
 		);
 	}
 
-	const matchingAsset = stats.assets.find((asset) => {
-		// Make sure to only look at js files and not source maps (assumes source maps don't end in .js)
+	// An asset may contain more than one chunk. In that case the size of
```
Comment (alexvy86) [inline build-tools/packages/bundle-size-tools/src/utilities/getChunkParsedSize.ts:17]: Truncated sentence.

**Review by alexvy86 — COMMENTED** [review]
For my education, what exactly are the assets and chunks here? I tried to map them to my understanding of bundling and failed 😄 .

**Comment by tylerbutler** [comment]
> For my education, what exactly are the assets and chunks here? I tried to map them to my understanding of bundling and failed 😄 .

I'm not much of an expert, but there are definitions here: https://webpack.js.org/glossary/

**Comment by tylerbutler** [comment]
This was folded into #16605.

**Pull request closed without merging**